MCprep_addon/conf.py

```diff
@@ -69,7 +69,7 @@ class Engine(enum.Enum):
 class MCprepEnv:
 	def __init__(self):
 		self.data = None
-		self.json_data = None
+		self.json_data: Optional[Dict] = None
```

Comment:
My POV is that if we add typing, we should do so for the entire definition (making data optional = None above as well, and also hint what the non-optional form is). Good improvement either way 👍